### ISSUE TITLE
Readme.md

### ISSUE THREAD

**Issue by gnai** [Qqi-HE/DeepChorus#1]
Hey, seems like an interesting repo. 
Can you give us a bit more details on how to run this and what's the input/output/parameters please? 
Thanks.

**Comment by Qqi-HE** [comment]
Thanks for your interest! Here is a quick summary:

**Input**: Raw audio files (MP3, WAV, etc.) of any length
**Output**: A binary vector indicating chorus (1) vs non-chorus (0) regions in the audio

**Main Parameters** (in `constant.py`):
- `feature_files` / `train_feature_files` / `test_feature_files`: paths to extracted mel-spectrogram `.joblib` files
- `train_annotation_file` / `valid_annotation_file` / `test_annotation_files`: paths to label `.joblib` files (format: `dict = {"song_name": [[start_time, end_time], ...], ...}`)
- `SHAPE`: input shape, default `(128, None, 1)` (128 mel bands, variable length)
- `CHUNK_SIZE`: temporal chunking size for the classifier
- `EPOCHS`, `LR`, `PATIENCE`: training hyperparameters

**Quick Start**:
1. Extract spectrograms: `python ./preprocess/extract_spectrogram.py`
2. Test: `python ./test.py -n DeepChorus -m Deepchorus_2021`
3. Train: `python ./train.py -n DeepChorus -m YourModelName`

Feel free to open another issue if you run into any problems!
